src/index.html

```diff
@@ -1,16 +1,57 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.cdnfonts.com/css/avenir" rel="stylesheet">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <!-- Big box -->
+    <form
+    action="#"
+    method="get"
+    class="box big"
+    data-qa="big">
```

Comment:
You need to check and fix it everywhere
<img width="1054" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/b0d10fe3-4ae7-4d37-a30a-cac31ff2a2d9">